mysql-config.yml.sample

```diff
@@ -8,6 +8,14 @@ instances:
         port: 3306
         username: newrelic
         password: <YOUR_SELECTED_PASSWORD>
+        # New users should leave this property as `true`, to identify the
+        # monitored entities as `remote`. Setting this property to `false` (the
+        # default value) is deprecated and will be removed soon, disallowing
+        # entities that are identified as `local`.
+        # Please check the documentation to get more information about local
+        # versus remote entities:
+        # https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/entity-definition.md
+        remote_monitoring: true
```

Comment:
should not be false by default?

Reply:
No, because we want new users to use remote entities. Previous installs will not have this option, thus, the integration will default to `false` (local entities).